examples/job_crud.py

```diff
@@ -12,6 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""
+Creates, updates, and deletes a Job object.
```

Comment:
s/Job/job

Reply:
I was just thinking about consistency, since Scott is using lower cased deployment, service, etc. in other files. Maybe those should be capitalized instead?